dashboard/config/android-modules/zip-kernel-headers.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Copyright 2021 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+# Zips kernel headers given a root kernel directory.
+
+# $1 Kernel common dir
+KERNEL_COMMON_DIR=$1
+KERNEL_HEADERS_TAR=${KERNEL_COMMON_DIR}/kernel-headers.tar.gz
+echo " Copying kernel headers to ${KERNEL_HEADERS_TAR}"
+pushd $KERNEL_COMMON_DIR
+  find arch include $KERNEL_COMMON_DIR -name *.h -print0               \
+          | tar -czf $KERNEL_HEADERS_TAR                     \
+            --absolute-names                                 \
+            --dereference                                    \
+            --transform "s,.*$KERNEL_COMMON_DIR,,"                     \
+            --transform "s,^,kernel-headers/,"               \
+            --null -T -
+popd
```

Comment:
Nit: unless you invoke this script with `source`, your cwd and directory stack are anyway gone after the execution is finished. So the script could be made a little simpler.

Reply:
Makes sense! I adjusted it